lib/functions/inline/arise_help.sh

```diff
@@ -9,29 +9,63 @@
 # arise_help
 
 arise_help() {
-cat <<EOF
-Arise, v$arise_version
-
-Welcome to Arise, a static site generator written in Bash
-
-Usage: 
-------
-bash arise build -[k][f] 
-        # Builds the entire site
-        available in all build modes:
-                -k: Keeps source files in output
-                -f: Force overwrite pre-existing output
-bash arise -[p|s|r][k][f]
-        # Builds only specific parts of the site
-        # Useful for testing purposes
-        mutually exclusive options:
-                -p: Build pages only mode
-                -s: Build sitemap only mode
-                -r: Build rss only mode
-
-------
-Please visit GitHub for more detailed info: 
-https://github.com/spectrasecure/arise
-
-EOF
+	echo "Version $arise_version"
+
+	cat <<"HELP"
+NAME
+
+	arise, a static site generator written in Bash
+
+SYNOPSIS
+
+	arise [<options>]...
+
+	Options must be given separately (e.g. "arise -f -k".)
```

Comment:
see above comments regarding keeping existing invocation syntax.
```suggestion
	arise build [<options>]...

	Options must be given separately (e.g. "arise build -f -k".)
```